sdk/appservice/mgmt/src/main/java/com/azure/management/appservice/implementation/FunctionAppImpl.java

```diff
@@ -92,8 +96,9 @@
         SiteLogsConfigInner logConfig,
         AppServiceManager manager) {
         super(name, innerObject, siteConfig, logConfig, manager);
-        functionAppKeyServiceHost = manager.restClient().getBaseUrl().toString();
-        functionAppKeyService = RestProxy.create(FunctionAppKeyService.class, manager.restClient().getHttpPipeline());
+        // TODO: confirm host url
```

Comment:
host url should be correct, same resource manager endpoint..

Prefer re-run some live appservice sample tests before merge.